### DISCUSSION TITLE
How To Add Ready To Review Lable

### DISCUSSION THREAD

**Discussion by shayan-yousefi** [goravel/goravel#465] [Q&A] [answered]
Hey there,

I've created a pull request, but it seems I don't have the necessary permissions to add labels like "Ready to Review." Could someone with the appropriate access help with this?

pr: https://github.com/goravel/framework/pull/591

**Comment by hwbrzzl** [comment, marked as answer]
Yes, may need permission, you can leave a comment when it's ready.
